tests/phpunit/BooleanValueTest.php

```diff
@@ -52,16 +52,21 @@ public function testGetValue( BooleanValue $boolean, array $arguments ) {
 		$this->assertEquals( $arguments[0], $boolean->getValue() );
 	}
 
-	public function testSerializationStability(): void {
-		$this->assertSame(
-			'C:23:"DataValues\BooleanValue":1:{1}', // Obtained with PHP 8.1.5, tested down to PHP 7.2
-			serialize( new BooleanValue( true ) )
-		);
+	/** @dataProvider instanceWithHashProvider */
+	public function testGetHashStability( BooleanValue $string, string $hash ) {
+		$this->assertSame( $hash, $string->getHash() );
+	}
 
-		$this->assertSame(
-			'C:23:"DataValues\BooleanValue":1:{0}', // Obtained with PHP 8.1.5, tested down to PHP 7.2
-			serialize( new BooleanValue( false ) )
-		);
```

Comment:
Why are these tests removed? Does the serialization change? And if so, is that something that cannot be avoided, and do you think it should constitute a breaking change?